samples/openthread/cli/harness/overlay-cert.conf

```diff
@@ -1,12 +1,12 @@
-# Disable defaults
-CONFIG_NCS_SAMPLES_DEFAULTS=n
-
-# Disable logging
-CONFIG_NET_LOG=n
-CONFIG_LOG=n
-CONFIG_NET_STATISTICS=n
-CONFIG_OT_COMMAND_LINE_INTERFACE_LOG_LEVEL_DBG=n
+# Move logging to RTT
 CONFIG_BOOT_BANNER=n
+CONFIG_LOG_MINIMAL=n
+CONFIG_SHELL_LOG_BACKEND=n
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_OPENTHREAD_LOG_LEVEL_DEBG=y
+CONFIG_LOG_STRDUP_BUF_COUNT=24
```

Comment:
How does this and line below impact RAM usage?

Reply:
+1328 bytes. 76 % RAM is used in this case for nrf52833.

I would say we have room enough to enable the more detailed logging level and cut it in the future if needed. This allows us to have some idea of the maximum requirements.